### PR TITLE
[Bugz-769] :: Cal app new intro

### PR DESCRIPTION
**Pull request by MiladNazeri**
https://highfidelity.atlassian.net/browse/BUGZ-769

### CODE REVIEW

File: usefulUtilities/hiFiCalendar/hifiWebApp/public/meetingRoom_ui.html [hunk 1]
```diff
@@ -3,13 +3,47 @@
...
+            <pre id="content" style="white-space: pre-wrap;"></pre>
+            <div id="background1">
+                <h1>Meeting Room Connector</h1>         
+                <p><h2>Connect your virtual meeting rooms to your Google Calendar for easy space management.</h2>
+            </div>
+            <div id="h1TopRight"></div>
+            <div id="h1BottomLeft"></div>
+            <div id="appInfo">
+                <p>
+                    This app shows google calendar schedules in your space.  It helps your company reserve meeting rooms and displays future events.
```
Comment (zfox23) [inline usefulUtilities/hiFiCalendar/hifiWebApp/public/meetingRoom_ui.html]:
```suggestion
                    This app links the meeting room schedules on your Google Calendar to meeting rooms in your virtual HQ, allowing you to see current and future meetings without leaving your headquarters. 
```

File: usefulUtilities/hiFiCalendar/hifiWebApp/public/meetingRoom_ui.html [hunk 2]
```diff
@@ -3,13 +3,47 @@
...
+                <p><h2>Connect your virtual meeting rooms to your Google Calendar for easy space management.</h2>
+            </div>
+            <div id="h1TopRight"></div>
+            <div id="h1BottomLeft"></div>
+            <div id="appInfo">
+                <p>
+                    This app shows google calendar schedules in your space.  It helps your company reserve meeting rooms and displays future events. 
+                </p>
+                <p>
+                    This app will access your available calendars and grab the latest information from the resources you choose to connect.
```
Comment (zfox23) [inline usefulUtilities/hiFiCalendar/hifiWebApp/public/meetingRoom_ui.html]:
```suggestion
                    This app will have read-only access to your Google Calendars, and will parse the latest event information from the specific calendars that you choose to connect.  
```

File: usefulUtilities/hiFiCalendar/hifiWebApp/public/meetingRoom_ui.html [hunk 3]
```diff
@@ -3,13 +3,47 @@
...
+            <div id="appInfo">
+                <p>
+                    This app shows google calendar schedules in your space.  It helps your company reserve meeting rooms and displays future events. 
+                </p>
+                <p>
+                    This app will access your available calendars and grab the latest information from the resources you choose to connect.  
+                </p>
+
+                <p>
+                    The only data that will be stored temporarily are the next 2 meetings currently scheduled and that information is deleted when they are in the past.
```
Comment (zfox23) [inline usefulUtilities/hiFiCalendar/hifiWebApp/public/meetingRoom_ui.html]:
```suggestion
                    After you connect your Calendars, the only data that will be temporarily stored is information regarding the next 2 meetings currently scheduled on that calendar. That data is permanently deleted when the events have concluded.
```

File: usefulUtilities/hiFiCalendar/hifiWebApp/public/meetingRoom_ui.html [hunk 4]
```diff
@@ -3,13 +3,47 @@
...
+                </p>
+                <p>
+                    Only rerun this app of you need to connect to different calendar resources.
```
Comment (zfox23) [inline usefulUtilities/hiFiCalendar/hifiWebApp/public/meetingRoom_ui.html]:
```suggestion
                    Only rerun this app if you need to connect your virtual meeting rooms to different Google Calendars.
```

**Comment by zfox23** [comment]
image

Does this happen in the hifi app?

i also noticed that the page looks strange on desktop - the top container is left-aligned and has a fixed width, while it seems as though the content below doesn't follow the same rules?

this isn't a problem in and of itself (i only care that it looks good in-world), but usually speaks to some issue in the code

image

File: usefulUtilities/hiFiCalendar/hifiWebApp/public/css/styles.css [hunk 5]
```diff
@@ -209,6 +209,24 @@ footer {
...
 
 /*** PAGES ***/
+#introPage {
+  width: 100%;
+}
+
+#appInfo {
+  position: absolute;
+  width: 90%;
+  height: 400px;
```
Comment (zfox23) [inline usefulUtilities/hiFiCalendar/hifiWebApp/public/css/styles.css]: you can remove this declaration, right?

**Comment by MiladNazeri** [comment]
image
^--- Change from 14.5 to 12.5px

**Comment by salinacsoto** [comment]
"This app links the meeting room schedules on your Google Calendar to meeting rooms in your virtual HQ, allowing you to see current and future meetings without leaving your headquarters" missing period at the end of that sentence.

**Comment by salinacsoto** [comment]
cant

Learn How leads nowhere and I can't get back to linking my calendars since there is no back button. I have to close the app and open it again so I can reach that page again.

**Pull request merged**